ci-operator/step-registry/baremetal/lab/pre/load-balancer/run/baremetal-lab-pre-load-balancer-run-commands.sh

```diff
@@ -83,13 +83,19 @@ for dev in "${devices[@]}"; do
     /sbin/dhclient -v \
     -pf "/var/run/dhclient.$interface.pid" \
     -lf "/var/lib/dhcp/dhclient.$interface.lease" "$interface"
-
-  # nsenter -m -u -n -i -p -t "$(podman inspect -f '{{ .State.Pid }}' "haproxy-$CLUSTER_NAME")" \
-  #   /sbin/dhclient -v -N \
-  #   -pf "/var/run/dhclient.$interface.v6.pid" \
-  #   -lf "/var/lib/dhcp/dhclient.$interface.v6.lease" "$interface"
 done
 
+if [ x"${DISCONNECTED}" != x"true" ]; then
```

Comment:
Can we do this within the if, with no duplication of the nsenter line and by running the command on any interface attached to the external bridge?

Reply:
I thought we said to run the command on any interface attached to the br-int ?

Reply:
rather than checking again for the DISCONNECTED variable, just check, within the `for`, whether the current iteration bridge is br-int, then use `$interface`.

Reply:
Done